src/features/DebugSession.ts

```diff
@@ -105,7 +114,7 @@ export class PickPSHostProcessFeature implements IFeature {
 
         if (this.waitingForClientToken) {
             this.clearWaitingToken();
-            return this.pickPSHostProcess();
```

Comment:
Shouldn't this still be called so that the menu will be displayed when the connection is finally established?